receiver/awsxrayreceiver/internal/udppoller/poller_test.go

```diff
@@ -381,19 +381,19 @@ func TestSocketGenericReadError(t *testing.T) {
 
 type mockNetError struct {
 	mockErrStr string
-	temporary  bool
+	timeout    bool
 }
 
 func (m *mockNetError) Error() string {
 	return m.mockErrStr
 }
 
 func (m *mockNetError) Timeout() bool {
-	return false
+	return m.timeout
 }
 
 func (m *mockNetError) Temporary() bool {
-	return m.temporary
+	return false
 }
```

Comment:
Can this be removed?

Reply:
if you are referring to the removing Temporary() function completely - mockNetError is implementing net.Error interface so it is required to implement the function even if it is not doing anything. Let me know if that is your intention.

Reply:
Ok. Sounds good